Update tutorial-filter-network-traffic.md (#111737)
Azure/azure-content: articles/virtual-network/tutorial-filter-network-traffic.md

A [network security group (NSG)](network-security-groups-overview.md) secures network traffic in your virtual network. 

> 1. From the Azure portal menu, select **+ Create a resource** > **Networking** > **Network security group**, or search for *Network security group* in the portal search box. (Network Security Group and not Network Security Group (classic))

Reviewer: ```suggestion
1. From the Azure portal menu, select **+ Create a resource** > **Networking** > **Network security group**, or use the portal search box to search for **Network security group** (not *Network security group (classic)*).
```
